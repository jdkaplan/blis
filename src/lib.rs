pub mod bytecode;
pub mod compile;
pub mod parse;
pub mod runtime;
pub mod vm;

#[cfg(test)]
mod testing;
