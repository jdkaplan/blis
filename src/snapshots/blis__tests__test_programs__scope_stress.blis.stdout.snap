---
source: src/lib.rs
description: "let x = 1;\n\nfunc f(x, y, z) {\n    let x = x + 10;\n    let y = x + 20;\n\n    {\n        let z = z;\n        println(x, y, z);\n    }\n\n    println(x, y, z);\n}\n\nfunc main() {\n    f(x, 2, 3);\n\n    let x = 4;\n    f(x, 4, 5);\n}\n\nmain();\n"
input_file: test_programs/scope_stress.blis
---
11 31 3
11 31 3
14 34 5
14 34 5
